src/raftstore/store/metrics.rs

```diff
@@ -213,10 +213,19 @@ lazy_static! {
             "Bucketed histogram of rocksdb ingestion durations",
             exponential_buckets(0.005, 2.0, 20).unwrap()
         ).unwrap();
+
     pub static ref RAFT_INVALID_PROPOSAL_COUNTER_VEC: IntCounterVec =
         register_int_counter_vec!(
             "tikv_raftstore_raft_invalid_proposal_total",
             "Total number of raft invalid proposal.",
             &["type"]
         ).unwrap();
+
+    pub static ref RAFT_STORE_EVENT_TICK_DURATION: HistogramVec =
```

Comment:
The name "tick" may be confused with the function "tick" in raftstore, I suggest renaming it to "timeout".
BTW, I think the name should follow "tikv_raftstore_xxx".

Reply:
Now, I change the name to `tikv_raftstore_event_timeout`, is that acceptable?